core_backend/app/llm_call/llm_rag.py

```diff
@@ -0,0 +1,12 @@
+from ..configs.llm_prompts import ANSWER_QUESTION_PROMPT
+from .utils import _ask_llm
+
+
+def get_llm_rag_answer(question: str, faq_data: str) -> str:
```

Comment:
Note that the answer is at the moment in English. It does not ask it to generate it in the language identified. We could do that though!